libs/hmpb/src/ballot_templates/v3_nh_ballot_template.tsx

```diff
@@ -450,7 +450,7 @@ async function CandidateContest({
       writeInArea: {
         top: 0.3,
         right: -0.9,
-        bottom: 0.7,
+        bottom: compact ? 0.4 : 0.7,
```

Comment:
Interesting that v3 required a much more notable adjustment than v4! But regardless, the boxes in the PR description look great to me